src/components/pages/weather/elements/WeatherHeadingCell.js

```diff
@@ -2,13 +2,19 @@ import React, { Component } from 'react';
 
 export default class WeatherHeadingCell extends Component {
   render() {
+    const season = (this.props.season ? (this.props.season) : (null));
```

Comment:
We can be tricky here and use [JavaScript's OR operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators#Logical_OR_2) to return `this.props.season` if it exists and null if it doesn't:

`const season = this.props.season || null`